include/boost/geometry/geometries/variant.hpp

```diff
@@ -16,14 +16,19 @@
 
 
 #include <boost/variant/variant_fwd.hpp>
+#include <boost/mpl/front.hpp>
 
 
 namespace boost { namespace geometry {
 
 
 template <BOOST_VARIANT_ENUM_PARAMS(typename T)>
-struct point_type<boost::variant<BOOST_VARIANT_ENUM_PARAMS(T)> >
-    : point_type<T0>
+struct point_type<variant<BOOST_VARIANT_ENUM_PARAMS(T)> >
```

Comment:
Although we are not 100% consistent about this, external (to Boost.Geometry) libraries used in Boost.Geometry code are qualified with `boost::`. So here I would expect `boost::variant` (basically as it was before).